--- a/components/omnibox/browser/autocomplete_controller.cc
+++ b/components/omnibox/browser/autocomplete_controller.cc
@@ -555,12 +555,12 @@
   config_.provider_types &= ~OmniboxFieldTrial::GetDisabledProviderTypes();
   if (base::CommandLine::ForCurrentProcess()->HasSwitch("omnibox-autocomplete-filtering")) {
     const std::string flag_value = base::CommandLine::ForCurrentProcess()->GetSwitchValueASCII("omnibox-autocomplete-filtering");
-    config_.provider_types &= AutocompleteProvider::TYPE_KEYWORD | AutocompleteProvider::TYPE_SEARCH |
-        AutocompleteProvider::TYPE_HISTORY_URL | AutocompleteProvider::TYPE_BOOKMARK | AutocompleteProvider::TYPE_BUILTIN;
+    config_.provider_types &= static_cast<int>(AutocompleteProvider::Type::kKeyword) | static_cast<int>(AutocompleteProvider::Type::kSearch) |
+        static_cast<int>(AutocompleteProvider::Type::kHistoryUrl) | static_cast<int>(AutocompleteProvider::Type::kBookmark) | static_cast<int>(AutocompleteProvider::Type::kBuiltin);
     if (!flag_value.contains("bookmarks"))
-      config_.provider_types &= ~AutocompleteProvider::TYPE_BOOKMARK;
+      config_.provider_types &= ~static_cast<int>(AutocompleteProvider::Type::kBookmark);
     if (!flag_value.contains("chrome"))
-      config_.provider_types &= ~AutocompleteProvider::TYPE_BUILTIN;
+      config_.provider_types &= ~static_cast<int>(AutocompleteProvider::Type::kBuiltin);
   }
 
   // Providers run in the order they're added. Async providers should run first
