--- a/chrome/browser/enterprise/connectors/analysis/content_analysis_info.cc
+++ b/chrome/browser/enterprise/connectors/analysis/content_analysis_info.cc
@@ -15,6 +15,7 @@
 #include "components/enterprise/connectors/core/content_area_user_provider.h"
 #include "components/enterprise/connectors/core/features.h"
 #include "components/enterprise/connectors/core/reporting_utils.h"
+#include "components/safe_browsing/buildflags.h"
 #include "components/safe_browsing/core/common/features.h"
 #include "components/sessions/content/session_tab_helper.h"
 #include "components/signin/public/identity_manager/accounts_in_cookie_jar_info.h"
@@ -175,10 +176,12 @@
                     &download_item)
                     ->GetWeakPtr()
               : nullptr) {
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
   if (base::FeatureList::IsEnabled(safe_browsing::kEnhancedFieldsForSecOps)) {
     referrer_chain_ =
         safe_browsing::GetOrIdentifyReferrerChainForEnterprise(download_item);
   }
+#endif
   frame_url_chain_ = enterprise_connectors::CollectFrameUrls(
       web_contents_.get(), enterprise_connectors::DeepScanAccessPoint::DOWNLOAD,
       std::make_optional(
