--- a/chrome/browser/sync/sync_service_factory.cc
+++ b/chrome/browser/sync/sync_service_factory.cc
@@ -135,55 +135,6 @@
 
 namespace {
 
-// Returns TabGroupSyncService or null if the feature is disabled.
-// Tab group sync is enabled via separate feature flags on different platforms.
-tab_groups::TabGroupSyncService* GetTabGroupSyncService(Profile* profile) {
-  CHECK(profile);
-#if BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_MAC) || \
-    BUILDFLAG(IS_WIN)
-  tab_groups::TabGroupSyncService* service =
-      tab_groups::TabGroupSyncServiceFactory::GetForProfile(profile);
-  CHECK(service);
-  return service;
-#elif BUILDFLAG(IS_ANDROID)
-  const bool enable_tab_group_sync =
-      tab_groups::IsTabGroupSyncEnabled(profile->GetPrefs());
-  tab_groups::TabGroupTrial::OnTabGroupSyncEnabled(enable_tab_group_sync);
-  if (!enable_tab_group_sync) {
-    return nullptr;
-  }
-  tab_groups::TabGroupSyncService* service =
-      tab_groups::TabGroupSyncServiceFactory::GetForProfile(profile);
-  CHECK(service);
-  return service;
-#else
-  return nullptr;
-#endif  // BUILDFLAG(IS_CHROMEOS) || BUILDFLAG(IS_LINUX) || BUILDFLAG(IS_MAC) ||
-        // BUILDFLAG(IS_WIN)
-}
-
-// Returns TemplateURLService or nullptr if the feature is disabled. Currently,
-// only on Android we have a separate feature flag for template url sync, and
-// it's only enabled for LFF.
-TemplateURLService* GetTemplateURLService(Profile* profile) {
-  CHECK(profile);
-#if BUILDFLAG(IS_ANDROID)
-  if (base::FeatureList::IsEnabled(syncer::kSyncSearchEnginesAndroidLFF) &&
-      base::FeatureList::IsEnabled(omnibox::kOmniboxSiteSearch)) {
-    return TemplateURLServiceFactory::GetForProfile(profile);
-  }
-  return nullptr;
-#else
-  return TemplateURLServiceFactory::GetForProfile(profile);
-#endif  // BUILDFLAG(IS_ANDROID)
-}
-
-autofill::AddressDataManager* GetAddressDataManager(Profile* profile) {
-  auto* pdm =
-      autofill::PersonalDataManagerFactory::GetForBrowserContext(profile);
-  return pdm ? &pdm->address_data_manager() : nullptr;
-}
-
 std::unique_ptr<KeyedService> BuildSyncService(
     std::optional<syncer::CreateHttpPostProviderFactory>
         create_http_post_provider_factory_for_test,
