--- a/extensions/browser/updater/update_service.cc
+++ b/extensions/browser/updater/update_service.cc
@@ -106,15 +106,10 @@
                                      const update_client::CrxUpdateItem& item) {
   DCHECK_CALLED_ON_VALID_THREAD(thread_checker_);
 
-  // Custom attributes can only be sent for NOT_UPDATED/UPDATE_FOUND events.
-  bool should_perform_action_on_omaha_attributes = false;
-
   switch (item.state) {
     case update_client::ComponentState::kUpToDate:
-      should_perform_action_on_omaha_attributes = true;
       break;
     case update_client::ComponentState::kCanUpdate:
-      should_perform_action_on_omaha_attributes = true;
       if (update_found_callback) {
         update_found_callback.Run(item.id, item.next_version);
       }
