diff --git a/chrome/browser/ui/toolbar/app_menu_model.cc b/chrome/browser/ui/toolbar/app_menu_model.cc
--- a/chrome/browser/ui/toolbar/app_menu_model.cc
+++ b/chrome/browser/ui/toolbar/app_menu_model.cc
@@ -432,24 +432,8 @@
           gfx::CHARACTER_BREAK)));
 }
 
 #if !BUILDFLAG(IS_CHROMEOS)
-std::u16string GetSyncSectionTitle(Profile* profile,
-                                   signin::IdentityManager* identity_manager) {
-  const AccountInfo account = GetAccountInfoFromProfile(profile);
-
-  if (IsSyncPaused(profile) || account.IsEmpty()) {
-    return l10n_util::GetStringUTF16(IDS_PROFILES_LOCAL_PROFILE_STATE);
-  }
-
-  if (signin_util::IsSigninPending(identity_manager)) {
-    return base::UTF8ToUTF16(account.GetEmail());
-  }
-
-  return l10n_util::GetStringFUTF16(
-      IDS_PROFILE_ROW_SIGNED_IN_MESSAGE_WITH_EMAIL,
-      {base::UTF8ToUTF16(account.GetEmail())});
-}
 
 class ProfileSubMenuModel : public ui::SimpleMenuModel,
                             public ui::SimpleMenuModel::Delegate {
  public:
