diff --git a/chrome/test/base/testing_browser_process.cc b/chrome/test/base/testing_browser_process.cc
index ebfccc15d7..cb11e0e8b4 100644
--- a/chrome/test/base/testing_browser_process.cc
+++ b/chrome/test/base/testing_browser_process.cc
@@ -30,7 +30,9 @@
 #include "chrome/browser/printing/print_job_manager.h"
 #include "chrome/browser/profiles/profile_manager.h"
 #include "chrome/browser/resource_coordinator/resource_coordinator_parts.h"
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
 #include "chrome/browser/safe_browsing/safe_browsing_service.h"
+#endif
 #include "chrome/browser/serial/serial_policy_allowed_ports.h"
 #include "chrome/browser/status_icons/status_tray.h"
 #include "chrome/common/chrome_paths.h"
@@ -476,10 +478,12 @@ StatusTray* TestingBrowserProcess::status_tray() {
   return status_tray_.get();
 }
 
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
 safe_browsing::SafeBrowsingService*
 TestingBrowserProcess::safe_browsing_service() {
   return sb_service_.get();
 }
+#endif
 
 WebRtcLogUploader* TestingBrowserProcess::webrtc_log_uploader() {
   return webrtc_log_uploader_.get();
@@ -794,10 +798,12 @@ TestingBrowserProcess::GetTestPlatformPart() {
   return platform_part_.get();
 }
 
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
 void TestingBrowserProcess::SetSafeBrowsingService(
     safe_browsing::SafeBrowsingService* sb_service) {
   sb_service_ = sb_service;
 }
+#endif
 
 void TestingBrowserProcess::SetWebRtcLogUploader(
     std::unique_ptr<WebRtcLogUploader> uploader) {
@@ -819,11 +825,6 @@ void TestingBrowserProcess::SetStatusTray(
 }
 
 #if !BUILDFLAG(IS_ANDROID)
-speech::SpeechRecognitionSmallExpertModelInstaller*
-TestingBrowserProcess::speech_recognition_small_expert_model_installer() {
-  return speech_recognition_small_expert_model_installer_.get();
-}
-
 void TestingBrowserProcess::SetComponentUpdater(
     std::unique_ptr<component_updater::ComponentUpdateService>
         component_updater) {
diff --git a/chrome/test/base/testing_browser_process.h b/chrome/test/base/testing_browser_process.h
index e8fe0af881..85fe9402ab 100644
--- a/chrome/test/base/testing_browser_process.h
+++ b/chrome/test/base/testing_browser_process.h
@@ -147,7 +147,9 @@ class TestingBrowserProcess
       std::unique_ptr<BackgroundModeManager> manager) override;
 #endif
   StatusTray* status_tray() override;
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
   safe_browsing::SafeBrowsingService* safe_browsing_service() override;
+#endif
   subresource_filter::RulesetService* subresource_filter_ruleset_service()
       override;
   BrowserProcessPlatformPart* platform_part() override;
@@ -204,8 +206,6 @@ class TestingBrowserProcess
   UsbSystemTrayIcon* usb_system_tray_icon() override;
   void set_usb_system_tray_icon_for_test(
       std::unique_ptr<UsbSystemTrayIcon> icon) override;
-  speech::SpeechRecognitionSmallExpertModelInstaller*
-  speech_recognition_small_expert_model_installer() override;
 #endif
   os_crypt_async::OSCryptAsync* os_crypt_async() override;
   void set_additional_os_crypt_async_provider_for_test(
@@ -220,7 +220,9 @@ class TestingBrowserProcess
 
   void SetMetricsService(metrics::MetricsService* metrics_service);
   void SetProfileManager(std::unique_ptr<ProfileManager> profile_manager);
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
   void SetSafeBrowsingService(safe_browsing::SafeBrowsingService* sb_service);
+#endif
   void SetVariationsService(variations::VariationsService* variations_service);
   void SetWebRtcLogUploader(std::unique_ptr<WebRtcLogUploader> uploader);
   void SetRulesetService(
@@ -338,7 +340,9 @@ class TestingBrowserProcess
       device_parental_controls_;
 #endif
 
+#if BUILDFLAG(SAFE_BROWSING_AVAILABLE)
   scoped_refptr<safe_browsing::SafeBrowsingService> sb_service_;
+#endif
   std::unique_ptr<subresource_filter::RulesetService>
       subresource_filter_ruleset_service_;
   std::unique_ptr<WebRtcLogUploader> webrtc_log_uploader_;
