--- a/chrome/browser/download/chrome_download_manager_delegate.cc
+++ b/chrome/browser/download/chrome_download_manager_delegate.cc
@@ -313,12 +313,14 @@
 }
 #endif  // SAFE_BROWSING_DOWNLOAD_PROTECTION
 
+#if BUILDFLAG(SAFE_BROWSING_DOWNLOAD_PROTECTION)
 // Returns true if the danger type is either FORCE_SAVE_TO_ONEDRIVE or
 // FORCE_SAVE_TO_GDrive.
 bool IsForceSaveToCloud(download::DownloadDangerType danger_type) {
   return danger_type == download::DOWNLOAD_DANGER_TYPE_FORCE_SAVE_TO_ONEDRIVE ||
          danger_type == download::DOWNLOAD_DANGER_TYPE_FORCE_SAVE_TO_GDRIVE;
 }
+#endif  // BUILDFLAG(SAFE_BROWSING_DOWNLOAD_PROTECTION)
 // Called asynchronously to determine the MIME type for |path|.
 std::string GetMimeType(const base::FilePath& path) {
 #if BUILDFLAG(IS_ANDROID)
