--- a/chrome/browser/search/search.cc
+++ b/chrome/browser/search/search.cc
@@ -14,18 +14,11 @@
 #include "build/build_config.h"
 #include "chrome/browser/profiles/profile.h"
 #include "chrome/browser/search_engines/template_url_service_factory.h"
-#include "chrome/browser/search_engines/ui_thread_search_terms_data.h"
-#include "chrome/browser/supervised_user/supervised_user_url_filtering_service_factory.h"
 #include "chrome/common/url_constants.h"
-#include "components/google/core/common/google_util.h"
 #include "components/search/ntp_features.h"
 #include "components/search/search.h"
 #include "components/search_engines/search_engine_type.h"
 #include "components/search_engines/template_url_service.h"
-#include "components/supervised_user/core/browser/supervised_user_preferences.h"
-#include "components/supervised_user/core/browser/supervised_user_url_filtering_service.h"
-#include "components/supervised_user/core/browser/supervised_user_utils.h"
-#include "components/supervised_user/core/common/buildflags.h"
 #include "content/public/browser/navigation_controller.h"
 #include "content/public/browser/navigation_entry.h"
 #include "content/public/browser/render_frame_host.h"
@@ -95,17 +88,6 @@
 
   NEW_TAB_URL_MAX
 };
-
-const TemplateURL* GetDefaultSearchProviderTemplateURL(Profile* profile) {
-  if (profile) {
-    TemplateURLService* template_url_service =
-        TemplateURLServiceFactory::GetForProfile(profile);
-    if (template_url_service) {
-      return template_url_service->GetDefaultSearchProvider();
-    }
-  }
-  return nullptr;
-}
 
 bool IsMatchingServiceWorker(const GURL& my_url, const GURL& document_url) {
   // The origin should match.
@@ -143,20 +125,6 @@
                                     IsMatchingServiceWorker(url, new_tab_url));
 }
 
-bool IsURLAllowedForSupervisedUser(const GURL& url, Profile& profile) {
-  if (!profile.IsChild()) {
-    return true;
-  }
-
-  supervised_user::SupervisedUserUrlFilteringService* url_filtering_service =
-      supervised_user::SupervisedUserUrlFilteringServiceFactory::GetForProfile(
-          &profile);
-  if (url_filtering_service->GetFilteringBehavior(url).IsBlocked()) {
-    return false;
-  }
-  return true;
-}
-
 // Used to look up the URL to use for the New Tab page. Also tracks how we
 // arrived at that URL so it can be logged with UMA.
 struct NewTabURLDetails {
