--- a/components/security_interstitials/content/ssl_blocking_page_base.cc
+++ b/components/security_interstitials/content/ssl_blocking_page_base.cc
@@ -8,16 +8,7 @@
 #include "components/security_interstitials/core/controller_client.h"
 #include "components/security_interstitials/core/metrics_helper.h"
 #include "components/strings/grit/components_strings.h"
-#include "components/user_prefs/user_prefs.h"
-#include "content/public/browser/browser_context.h"
-#include "content/public/browser/web_contents.h"
 #include "ui/base/l10n/l10n_util.h"
-
-namespace {
-PrefService* GetPrefs(content::WebContents* web_contents) {
-  return user_prefs::UserPrefs::Get(web_contents->GetBrowserContext());
-}
-}  // namespace
 
 SSLBlockingPageBase::SSLBlockingPageBase(
     content::WebContents* web_contents,
