diff --git a/chrome/browser/chrome_content_browser_client.cc b/chrome/browser/chrome_content_browser_client.cc
--- a/chrome/browser/chrome_content_browser_client.cc
+++ b/chrome/browser/chrome_content_browser_client.cc
@@ -434,6 +434,7 @@
 #include "services/network/public/mojom/url_loader_factory.mojom.h"
 #include "services/network/public/mojom/web_transport.mojom.h"
 #include "services/network/public/mojom/websocket.mojom.h"
+#include "services/screen_ai/buildflags/buildflags.h"
 #include "third_party/blink/public/common/chrome_debug_urls.h"
 #include "third_party/blink/public/common/features.h"
 #include "third_party/blink/public/common/loader/url_loader_throttle.h"
@@ -8316,6 +8317,7 @@
                                    soda_language_pack_path.value()));
     return true;
   }
+#if BUILDFLAG(ENABLE_SCREEN_AI_SERVICE)
   if (sandbox_type == sandbox::mojom::Sandbox::kScreenAI) {
     // ScreenAI service needs read access to ScreenAI component binary path to
     // load it.
@@ -8330,6 +8332,7 @@
         sandbox::policy::kParamScreenAiComponentPath,
         screen_ai_binary_path.value());
   }
+#endif  // BUILDFLAG(ENABLE_SCREEN_AI_SERVICE)
   if (sandbox_type == sandbox::mojom::Sandbox::kOnDeviceTranslation) {
     auto translatekit_binary_path =
         on_device_translation::ComponentManager::GetInstance()
